### ISSUE TITLE
Add a download link for a minor uv version

### ISSUE THREAD

**Issue by danielhollas** [astral-sh/uv#4109]
Right now when installing uv via URL, one can either install the latest version

```console
curl -LsSf https://astral.sh/uv/install.sh | sh
```

or a specific version

```console
curl -LsSf https://astral.sh/uv/0.2.9/install.sh | sh
```

Given that uv now has official version policy, it would be nice to be able to specify a major and minor version, without pinning the patch version. Something like:

```
curl -LsSf https://astral.sh/uv/0.2/install.sh | sh
```

**Comment by zanieb** [comment]
(We should do this)

**Comment by samypr100** [comment]
This would be arguably nice to extend docker tags too

**Comment by zanieb** [comment]
Good point! I had some fun with this at my last job... https://github.com/PrefectHQ/prefect/blob/4da5da207b126639998917cb1e246f7b15484405/.github/workflows/docker-images.yaml#L110-L116

**Comment by nikpivkin** [comment]
I found this command in the release `curl --proto ‘=https’ --tlsv1.2 -LsSf https://github.com/astral-sh/uv/releases/download/0.5.8/uv-installer.sh | sh`. Why is it not documented [here](https://docs.astral.sh/uv/getting-started/installation/#standalone-installer)?

**Comment by zanieb** [comment]
@nikpivkin that's documented at https://docs.astral.sh/uv/getting-started/installation/#github-releases

**Comment by mynewestgitaccount** [comment]
Until this is added, does anyone know a way to install the latest patch in a minor version without going through pip?

Context: I am installing uv in an Azure Pipeline like `pip install uv~=0.x.0` right now, but I would like to avoid using pip if possible. Searching for alternatives is how I found this issue, actually. The proposed download link one-liner seems very natural.
